docs/guides/hybrid-graphics.md

```diff
@@ -7,9 +7,36 @@ This has been tested on the MacBookPro16,1 and the MacBookPro15,1. The 15,3 and
 ## Issues
 
 1. Resume after suspend is broken, as the GMUX (graphics multiplexer) doesn't connect the iGPU to the display after resuming. For this to be fixed, a Linux driver for `acpi:APP000B:GPUC:` needs to be written (macOS uses AppleMuxControl2.kext). The extra battery life may make this a worthwhile trade-off (about 3 hours to almost 6 hours on a MacBookPro16,1)
+
 2. If using `DRI_PRIME=1` on programs causes system crashes, with "[CPU CATERR](https://gist.github.com/Redecorating/956a672e6922e285de83fdd7d9982e5e#gistcomment-3719941)" problem reports in macOS, disable dynamic power management with the `amdgpu.dpm=0` kernel argument, or `echo high | sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level`.
+
 3. If apple-set-os is loaded, the iGPU will control display brightness, and if the iGPU isn't the boot gpu, the i915 Intel graphics driver will not load, and the display brightness cannot be changed (The exception to this is sometimes when rebooting from macOS Recovery etc, i915 loads fine).
 
+4. Some users with MacBookPro16,1 and AMD gpu have reported issues like AMD gpu uses too much power with excessive high temperatures under normal conditions or sudden high speed fan noise then instant system shut down, getting the T2 chip reset.
+
+    Posible workarounds:
+
+    1. Set iGPU as main gpu.
+
+    2. Set AMD gpu Dynamic Power Management from auto to low.
```

Comment:
maybe add "(or high)" here because that's also a way to stop the issue

Reply:
I would prefer to avoid  high as can lead to thermal issues. In my case I have low by the default and only high when playing games. Some times high can lead to crashes. I found low as most safer setting if you are with amdgpu.

Reply:
ok Im going to add change